DashClothes/settings.py

```diff
@@ -28,7 +30,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['*']
+ALLOWED_HOSTS = []
```

Comment:
@rinkishivani why empty ALLOWED_HOSTS = [] ?

Reply:
added * in allowed hosts.